lib/ruby/lambdas/core/strict_mode/disable.rb

```diff
@@ -0,0 +1,13 @@
+[:String!, :Numeric!].each do |name|
+  Kernel.remove_method(name) if Kernel.respond_to?(name)
+end
```

Comment:
I think it's good to add a description or something like to show that this methods will be overwritten by new ones. I don't know if we add this to the readme, or some king of documentation

Reply:
@leandrotk what you think about this short explanation:
```ruby
# These Kernel methods can be overridden by 
# new ones if the strict mode was enabled. e.g:
# require "ruby/lambdas/strict-mode"
```

Reply:
It's great!